cpp/include/cuvs/neighbors/ivf_pq.hpp

```diff
@@ -470,17 +498,22 @@ struct index : cuvs::neighbors::index {
   raft::device_vector_view<const uint32_t, uint32_t, raft::row_major> list_sizes() const noexcept;
 
   /** Cluster centers corresponding to the lists in the original space [n_lists, dim_ext] */
-  raft::device_matrix_view<float, uint32_t, raft::row_major> centers() noexcept;
   raft::device_matrix_view<const float, uint32_t, raft::row_major> centers() const noexcept;
+  /** Setter for the centers [n_lists, dim_ext] or [n_lists, dim] */
+  void update_centers(raft::resources const& handle,
```

Comment:
How exactly is this different from the `set_centers` helper that we currently have? Can we re-use `set_centers` in places where you are calling `update_centers`?

Reply:
Sounds good! Feel free to mark this one as resolved.